test/decl/protocol/objc_protocol_inheriting.swift

```diff
@@ -0,0 +1,20 @@
+// RUN: %target-run-simple-swift | %FileCheck %s
```

Comment:
This directory should not contain executable tests. You can move this test to test/Interpreter instead.